### DISCUSSION TITLE
Are endpoints from Enmeshed platform V1 no more available?

### DISCUSSION THREAD

**Discussion by hugotiburtino** [nmshd/feedback#18] [Q&A] [answered]
Hi,
~I was trying to reactivate our old PoC for a demo and I just noticed that our connection is getting the following error message~
```console
HttpServer - POST /api/v1/RelationshipTemplates/Own/.../Token 404 -- 37ms
```
~I'm just wondering: the v1 endpoints aren't available anymore, are they?~

edit: sorry, our problem

**Comment by jkoenig134** [comment, marked as answer]
Hi Hugo,

you told that's a problem on your side, but I want to clarify for other people that may get the same errors.

We announced the [GA of Enmeshed V2 a while ago](https://enmeshed.eu/blog/road-to-enmeshed-v2), and the Connector `latest` image is now pointing on `v3.x.x` instead of before `v2.x.x`.

As announced we will wind down the Enmeshed V1 support until 1st of December. Make sure to [migrate your application to the Enmeshed V2 Connector](https://enmeshed.eu/integrate/connector-migration-v2) until then.
